src/components/mes-signature-pad/mes-signature-pad.html

```diff
@@ -0,0 +1,18 @@
+<div class="not-valid-header">
+    <span class="mes-header-md" *ngIf="isvalid === false && notValidHeaderText && required">
+        <label id="not-valid-header-label" class="not-valid-text">{{notValidHeaderText}}</label>
+    </span>
+</div>
+<div class="mes-signature-pad" (blur)="checkAndSetValidation()" [class.is-not-valid]="isvalid === false  && required">
+    <signature-pad (onEndEvent)="drawComplete()"></signature-pad>
+    <button (click)="clear()" class="retry-button" [disabled]="isvalid === false ">
```

Comment:
could disabled be !isValid instead of === false

Reply:
I want null to effectively be untouched. only false to active the validation styles